### PR TITLE
Description changes

### PR DESCRIPTION
**Pull request by InsanityPrelude**
I've been bothered for about a year or so about a number of descriptions for everything from research to recipes to individual machines.  It's rarely anything big, just little grammar errors or inconsistent use of tenses or what have you.  There were a small number of items in the Floor and Lights category which randomly said "Added by Project RimFactory" though literally nothing else in the mod did, and it just generally seemed like the descriptions were added in over multiple years by multiple authors with wildly divergent quality.

I didn't want to completely redo and rename everything to try to match a consistent theme like the complex references to automation and such in the primary research projects, but I went in and gave a solid editing pass to everything.  I tweaked some descriptions more than others where what was oddly un-descriptive or unusual compared to the rest of the work, but it should all match the general flavor of RimFactory.

Feel free to shout at me on Discord if you hate something, and use any, all, or none of these.

### CODE REVIEW

File: Defs/ThingDefs_Buildings/Buildings_SAL_Like.xml [hunk 1]
```diff
@@ -185,7 +185,7 @@
   <ThingDef ParentName="PRF_SAL_Base">
     <defName>PRF_TheArtMachine</defName>
     <label>The Art Machine(S.A.L)</label>
-    <description>Pumps out art like a hose pumps out water.\n\nStats:\nBase: 3\n Crafting: 7\nArtistic: 8\nSpeed: 1</description>
+    <description>A specially designed Auto Crafter which uses heuristic algorithms to produce moderately skilled art at a pace no human could ever match.\n\nStats:\nBase: 3\n Crafting: 7\nArtistic: 8\nSpeed: 1</description>
```
Comment (lilwhitemouse) [inline Defs/ThingDefs_Buildings/Buildings_SAL_Like.xml:188]: Do we want to put back in the whimsical image of a hose pumping out art?

**Comment by lilwhitemouse** [comment]
Looks very solid; my only thought is to ask if we want to keep the "hose" and "volcano" imagery for the SAL art machines.

I can look at merging the conflicts tomorrow.

**Comment by zymex22** [comment]
but? im confused.... i just uploaded new descriptions aleady right? from Shipidge 
https://github.com/zymex22/Project-RimFactory-Revived/issues/473
//
https://github.com/zymex22/Project-RimFactory-Revived/commit/92a144035c51edf93495f91e7c615d8908e370dd

what are those in PR then?

**Comment by zymex22** [comment]
i see you have done some more than shipi did eq. recipes.
shame we got a bit of double work done, but any case THANKS for great work, but fixing up our text to both of you.
This been long overdue, to get a complete text overhaul

**Review by zymex22 — COMMENTED** [review]
these 7 files are not changed in latest update and these could were well just go in as is.
rest will need some more investigation, which are also why they conflicts as i just changed those.

**Comment by Sn1p3rr3c0n** [comment]
closing as the changes have been merged with #518

**Pull request closed without merging**